algorithms/ar-binsrt/javascript/binarySort.js

```diff
@@ -0,0 +1,45 @@
+function binarySort(item, sortedList, low = 0, high = sortedList.length - 1) {
+  if (low == high) {
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318101)